Archipelago.HollowKnight/SlotData/SlotDataExtract.cs

```diff
@@ -18,5 +19,16 @@ internal static class SlotDataExtract
             Archipelago.Instance.LogDebug($"Successfully read object from SlotData: {jarr}");
             return extractedObject;
         }
+
+        public static Dictionary<string, Dictionary<string, int>> ExtractLocationCostsFromSlotData(object v)
+        {
+            Dictionary<string, Dictionary<string, int>> ret = new();
+            var jobj = ExtractObjectFromSlotData<Dictionary<string, JObject>>(v);
+            foreach (var key in jobj.Keys)
+            {
+                ret[key] = jobj[key].ToObject<Dictionary<string, int>>();
```

Comment:
```suggestion
                ret[key] = ExtractObjectFromSlotData<Dictionary<string, int>>(jobj[key]);
```
`ExtractObjectFromSlotData` just has some null safety that's nice to have around, so might as well use it.

Reply:
We can do that, but it ends up logging a *ton* of garbage to the log.  Or maybe it's time to remove some of that debug logging (along the item placement logging)

Reply:
Debug logging being chatty isn't a bad thing.